site/en/tutorials/keras/regression.ipynb

```diff
@@ -1052,7 +1052,7 @@
         "history = dnn_horsepower_model.fit(\n",
         "    train_features['Horsepower'], train_labels,\n",
         "    validation_split=0.2,\n",
-        "    verbose=0, epochs=100)"
+        "    verbose=0, epochs=100, validation_split=0.2)"
```

Comment:
@BichengWang the `validation_split` arg (value set to `0.2`) is already present in `model.fit`, so I think your commit introduces a duplicate. Can you double check please?

Before:
```diff
        "history = dnn_horsepower_model.fit(\n",
         "    train_features['Horsepower'], train_labels,\n",
         "    validation_split=0.2,\n",
         "    verbose=0, epochs=100)"
```

After your commit:
- `validation_split=0.2` appears twice:
```diff
        "history = dnn_horsepower_model.fit(\n",
         "    train_features['Horsepower'], train_labels,\n",
         "    validation_split=0.2,\n",
         "    verbose=0, epochs=100, validation_split=0.2)"
```

Reply:
You are right. Sorry to miss this part. Will close the ticket.